Sign for JabberFR

linkmauve opened the pull request:
We are already applying most of these measures, and will enforce blocking of non-compliant domains in July.

linkmauve committed:
Sign for both main domains of JabberFR

ge0rg commented:
Thanks very much.

I've never heard of apinc.org, and according to its website it's dead since 2016. I've stated in other tickets that co-hosted services should only list their primary domain name, so it would be unfair to let you have both domains listed, especially since one of them lacks a homepage and own branding. Please limit the PR to jabber.fr :)

Also, just for the forms sake, I'd like to have a sign-off from Mathieu, either here in the PR or via XMPP. Thanks!

linkmauve committed:
Remove the APINC domain we host for their users

APINC stopped providing web hosting but we continue to host their Jabber service. It is thus considered as a secondary domain of ours, so removing it from this list.

linkmauve commented:
I removed im.apinc.org, it still has https://jabber.apinc.org/ though but that one is getting progressively merged into https://jabberfr.org/ so that’s fair.

ge0rg commented:
Thanks, squashed and merged in 90ae51f7c264b88fc84c8edc3c8adc8939b2b077.
